.github/workflows/main.yml

```diff
@@ -68,6 +87,35 @@ jobs:
           GATSBY_EXPERIMENTAL_QUERY_CONCURRENCY: 8 # default is 4, going too high on the 2-core github actions machines will fire back
           GATSBY_CPU_COUNT: logical_cores # use hyperthreading if available to statistically better saturate the cores
 
+      - name: GitHub Slug Action
+        uses: rlespinasse/github-slug-action@v4
+        with:
+          slug-maxlength: 52
+
+      - name: Deploy preview to commercetools-docs-kit vercel via CLI
```

Comment:
Chamge to the same system we use in the other repos now,  the unmaintained action had issues with with node 18 (it seemed) and its vercel version got old too